tests/unit/job/test_genericJob.py

```diff
@@ -115,7 +115,10 @@ def test_compress_file_list(self):
         )
         with contextlib.redirect_stdout(io.StringIO(newline=os.linesep)) as f:
             ham.files.file_to_compress.tail()
-        self.assertEqual(f.getvalue().replace("\r", ""), "content: file_to_compress\n")
+        self.assertEqual(
+            f.getvalue().replace("\r", "").replace("\n", ""),
```

Comment:
```suggestion
            f.getvalue().replace("\n", ""),
```

Re-reading the original issue on the monty repo, this may have been a work around for a bug from them on windows.  I'll commit this and revert in case the test fails, otherwise lgtm.